schemas/rum/_parent-view-schema.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "rum/_parent-view-schema.json",
+  "title": "ParentViewSchema",
+  "type": "object",
+  "description": "Parent view schema for views that are nested (webviews in mobile)",
+  "properties": {
+    "parent_view": {
+      "type": "object",
+      "description": "Parent view properties (view wrapping the current view)",
+      "required": ["id", "source"],
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "ID of the parent view",
+          "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$",
+          "readOnly": true
+        },
+        "source": {
+          "type": "string",
+          "description": "Source of the parent view",
+          "enum": ["android", "ios", "browser", "flutter", "react-native", "roku"],
```

Comment:
I see already too many 

`      "enum": ["android", "ios", "browser", "flutter", "react-native", "roku"],
`

Duplicated enums.

Suggestion:
create a shared enum

Reply:
not sure our schema generator supports that now honestly. I would rather not add this new constraint to it.